src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -133,9 +136,15 @@ export class DynamicFacetManager extends Component {
     this.bind.onRootElement(QueryEvents.querySuccess, (data: IQuerySuccessEventArgs) => this.handleQuerySuccess(data));
   }
 
-  private handleAfterComponentsInitialization() {
+  private get anyDynamicFacets(): AnyDynamicFacet[] {
     const allDynamicFacets = this.bindings.searchInterface.getComponents<DynamicFacet>('DynamicFacet');
-    this.childrenFacets = allDynamicFacets.filter(dynamicFacet => this.element.contains(dynamicFacet.element));
+    const allDynamicFacetRanges = this.bindings.searchInterface.getComponents<DynamicFacetRange>('DynamicFacetRange');
```

Comment:
Im trying to think of a way this could be done in a more elegant manner, so that we don't end up with "forgetting" to add new facets here, and instead leverage the TypeScript compiler.

One way to do that would be to think of a function/property that all "Dynamic" facet will necessarily need to implement.

For example, let's say that all DynamicFacet had to implement a `onReorder` method, then it would be possible to do something like this : 

```
export interface IDynamicFacet {
  onReorder: () => void;
}

public isDynamicFacet(component: BaseComponent): component is IDynamicFacet {
    return 'onReorder' in component;
}

public getAllDynamicFacet() IDynamicFacet[] {
   return this.searchInterface.getAllComponents().filter(cmp => this.isDynamicFacet(cmp))
}
```

Now the "onReorder" function might not be good, but if you think if any property that would do the job, then I think it would be more future proof.

Reply:
For now they're all instances of DynamicFacet so we can use this